yang/ietf-l3vpn-ntw.yang

```diff
@@ -2345,6 +2353,38 @@ module ietf-l3vpn-ntw {
                     if-feature "vpn-common:bfd";
                     description
                       "Container for BFD.";
+                    leaf transmit_interval {
+                      type uint32;
+                      units microseconds;
+                      description
+                        "The minimum interval between transmission of BFD control
+                        packets that the operator desires. This value is advertised to
+                        the peer, however the actual interval used is specified by
+                        taking the maximum of transmit_interval and the
+                        value of the remote required-minimum-receive interval value.
+
+                        This value is specified as an integer number of microseconds.";
+                    }
+                    leaf receive_interval {
+                      type uint32;
+                      units microseconds;
+                      description
+                        "The minimum interval between received BFD control packets that
+                        this system should support. This value is advertised to the
+                        remote peer to indicate the maximum frequency (i.e., minimum
+                        inter-packet interval) between BFD control packets that is
+                        acceptable to the local system.";
+                    }
+                    leaf detection-multiplier {
+                      type uint8 {
+                        range "1..max";
```

Comment:
The range should justified by echoing this part from 5880, also a ref statement should be added

"   bfd.DetectMult
      ...  This variable MUST be a nonzero integer, ..."

Thanks